data/sql_setup/prepare_schedule_a.sql

```diff
@@ -92,27 +323,18 @@ begin
         -- visit here:
         -- https://www.postgresql.org/docs/current/static/plpgsql-statements.html#PLPGSQL-STATEMENTS-DIAGNOSTICS
         if FOUND then
-            two_year_transaction_period = cast(get_cycle(view_row.rpt_yr) as smallint);
-
-            if two_year_transaction_period >= start_year then
+            if view_row.election_cycle >= start_year then
                 delete from ofec_sched_a_queue_old where sub_id = view_row.sub_id;
-                insert into ofec_sched_a_queue_old values (view_row.*, timestamp, two_year_transaction_period);
+                insert into ofec_sched_a_queue_old values (view_row.*, current_timestamp, view_row.election_cycle);
```

Comment:
I was keeping the queue creation in there until I fixed the code for the aggregates.